sphinx/application.py

```diff
@@ -681,14 +681,19 @@ def add_javascript(self, filename):
             StandaloneHTMLBuilder.script_files.append(
                 posixpath.join('_static', filename))
 
-    def add_stylesheet(self, filename):
+    def add_stylesheet(self, filename, rel=None, title=None):
```

Comment:
I don't think passing arbitrary strings to `rel` is good API design.  This should be some kind of enum or, which I think would be totally sufficient for the use cases at hand, just an `alternate=bool` flag.